github/messages.go

```diff
@@ -43,80 +43,156 @@ const (
 	DeliveryIDHeader = "X-Github-Delivery"
 )
 
+type EventTyper interface {
+	EventType() EventType
+}
```

Comment:
Since this is not used anywhere, let's please delete it.
If someone finds this useful in their own code, then it is a simple matter to create it there instead of here.

Reply:
abcf17e3